### ISSUE TITLE
Create Title Scene

### ISSUE THREAD

**Issue by SilasReinagel** [EnigmaDragons/ProjectNeon#6]
Title Label
Start Game Button
Exit Game Button
Options Button

**Comment by SilasReinagel** [comment]
@0crat boost 1x

**Comment by SilasReinagel** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat boost 1x ([here](https://github.com/EnigmaDragons/ProjectNeon/issues/6#issuecomment-529689861))

@SilasReinagel There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@1bed83ee-72bd-4b90-b626-859c1bf71176, thread: pool-5-thread-5
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-boosts.xsl says: The boost for job "gh:enigmadragons/projectneon#6" exists in boosts.xml, but the job is absent in the WBS among 2 jobs.
```

[0.49.8](https://github.com/zerocracy/farm): CID: [844df2ff-1e58-40c1-903f-d6e10afced93](https://www.0crat.com/footprint/CN3V1M4AU/844df2ff-1e58-40c1-903f-d6e10afced93), Type: "Set boost"

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/EnigmaDragons/ProjectNeon/issues/6#issuecomment-529689919))

@SilasReinagel Job [#6](https://github.com/enigmadragons/projectneon/issues/6) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Boosting tasks is against our principles, see [§15](http://www.zerocracy.com/policy.html#15): -10 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
The job [#6](https://github.com/enigmadragons/projectneon/issues/6) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CN3V1M4AU/eb6d7ebc-fb1d-4d93-80bd-f6eeb0a8d3db); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by SilasReinagel** [comment]
@0crat boost 1x

**Comment by 0crat** [comment]
> @0crat boost 1x  ([here](https://github.com/EnigmaDragons/ProjectNeon/issues/6#issuecomment-529693237))

@SilasReinagel Boost 1x was set for [#6](https://github.com/enigmadragons/projectneon/issues/6)

**Comment by 0crat** [comment]
Boosting tasks is against our principles, see [§15](http://www.zerocracy.com/policy.html#15): -10 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0pdd** [comment]
@SilasReinagel 3 puzzles [#39](https://github.com/EnigmaDragons/ProjectNeon/issues/39), [#40](https://github.com/EnigmaDragons/ProjectNeon/issues/40), [#41](https://github.com/EnigmaDragons/ProjectNeon/issues/41) are still not solved.

**Comment by 0crat** [comment]
Job was finished in 23 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#6](https://github.com/enigmadragons/projectneon/issues/6) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0pdd** [comment]
@SilasReinagel 2 puzzles [#40](https://github.com/EnigmaDragons/ProjectNeon/issues/40), [#41](https://github.com/EnigmaDragons/ProjectNeon/issues/41) are still not solved; solved: [#39](https://github.com/EnigmaDragons/ProjectNeon/issues/39).

**Comment by 0crat** [comment]
Job `gh:enigmadragons/projectneon#6` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Comment by 0pdd** [comment]
@SilasReinagel the puzzle [#40](https://github.com/EnigmaDragons/ProjectNeon/issues/40) is still not solved; solved: [#39](https://github.com/EnigmaDragons/ProjectNeon/issues/39), [#41](https://github.com/EnigmaDragons/ProjectNeon/issues/41).
